### DISCUSSION TITLE
Porter website created with Hextra

### DISCUSSION THREAD

**Discussion by zelfroster** [imfing/hextra#80] [Show and tell]
Firstly, Thank you so much for making this amazing theme ❤️.

[Porter](https://github.com/getporter/porter) team wanted to update their custom theme to some already maintained modern theme for their website made with Hugo, so they created this [issue](https://github.com/getporter/porter/issues/2885). So, I took this issue in my hands and started looking for themes and I found Hextra and It was awesome.

I used Hextra and this is their website now https://getporter.org .

![image](https://github.com/imfing/hextra/assets/93514460/36e9ed8c-5d3b-4b4b-b7e4-9a510d759090)

I wish this project grows, and more people would use this. Keep making this theme more awesome 🤗❤️.

**Comment by imfing** [comment]
@zelfroster Thank you for the kind words and for choosing Hextra for the Porter website! ❤️
It's incredibly rewarding to hear that you all like Hextra and the theme has already been used in a CNCF Sandbox project website! 
Feel free to let me know what I could help to make Porter's website even better.

It appears that Porter is the first open-source project to adopt this relatively new theme. I'm planning to add a showcase section on Hextra, and I'd be delighted to feature Porter there!

In the meantime, please don't hesitate to to send new issues, feature requests or discussions on our repository: your insights could help make Hextra even better for everyone.
